src/protocols/ssh/settings.h

```diff
@@ -178,6 +178,19 @@ typedef struct guac_ssh_settings {
      */
     bool sftp_disable_upload;
 
+    /**
+     * Whether line endings within the clipboard should be automatically
+     * normalized to Unix-style newline characters.
+     */
+    int normalize_clipboard;
+
+    /**
+     * Whether Unix-style newline characters within the clipboard should be
+     * automatically translated to CRLF sequences before transmission to the
+     * SSH server.
+     */
+    int clipboard_crlf;
```

Comment:
Is there any case where the terminal clipboard contents shouldn't be normalized to UNIX-style newlines? When data is pasted within a terminal emulator session, it's passed through to the remote end as if entered by the user. In all cases, if a user presses the "Enter" or "Return" key, only a single character will be sent:

https://github.com/apache/guacamole-server/blob/b20afa275a78ca4a54f4e088413e99c93901a3bc/src/terminal/terminal.c#L1675

If we selectively normalize or provide an option to transform these newlines into CRLF pairs, we'd be adding an additional character and producing a sequence that isn't normally produced by user input.